src/Microsoft.Health.Fhir.Azure/FhirServerBuilderAzureRegistrationExtensions.cs

```diff
@@ -41,12 +59,12 @@ public static IFhirServerBuilder AddKeyVaultSecretStore(this IFhirServerBuilder
             else
             {
                 fhirServerBuilder.Services.Add<KeyVaultSecretStore>((sp) =>
-                    {
-                        var tokenProvider = new AzureServiceTokenProvider();
-                        var kvClient = new KeyVaultClient(new KeyVaultClient.AuthenticationCallback(tokenProvider.KeyVaultTokenCallback));
+                {
+                    var tokenProvider = new AzureServiceTokenProvider();
+                    var kvClient = new KeyVaultClient(new KeyVaultClient.AuthenticationCallback(tokenProvider.KeyVaultTokenCallback));
 
-                        return new KeyVaultSecretStore(kvClient, new Uri(keyVaultConfig.Endpoint));
-                    })
+                    return new KeyVaultSecretStore(kvClient, new Uri(keyVaultConfig.Endpoint));
+                })
```

Comment:
nit: spacing seems a little weird here with .Singleton() starting with an extra tab compared to the previous line.